src/main/takeDetection/sentenceSimilarity.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * https://github.com/stephenjjbrown/string-similarity-js
+ * Calculate similarity between two strings
+ * @param {string} str1 First string to match
+ * @param {string} str2 Second string to match
+ * @param {number} [substringLength=2] Optional. Length of substring to be used in calculating similarity. Default 2.
+ * @returns Number between 0 and 1, with 0 being a low match score.
+ */
+export const getSimilarityScore = (
+  str1: string,
+  str2: string,
+  substringLength = 2
```

Comment:
What's this substringLength for?

Reply:
It indicates the length of the substring for each comparison, the higher the value the closer to the exact string match. I leave it here because It will be easier if we decide to use a tighter match later (guess not, I'll just move it to the inside function)